### PR TITLE
Update/simplify extract_connectivity_df 

### PR DESCRIPTION
**Pull request by romainFr**
It was failing for me because neuprint now does not return a pre (or post) field if the value is 0.

### CODE REVIEW

**Comment by romainFr** [comment]
That is, for the latest version of the server (13000), but that does not break the old version (11000)

**Comment by romainFr** [comment]
As a side note, there are  some functions failing in the new server (notably neuprint_connection_table) because of changes in the data model

**Comment by jefferis** [comment]
Thanks @romainFr. I will try to review as soon as possible. As you note there are a number of functions that have been broken by the new data model. Are you aware of any versioning scheme for neuPrint that might allow us to match the API client functions to the API version?

**Comment by romainFr** [comment]
Not that I am aware of. The "version" function in the API returns the same thing for both servers (0.1.0). I think their idea is that only the new data model will ever be public facing.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by jefferis** [comment]
Thanks for that note @romainFr. Versioning will apparently come when the API is a bit more stable.

File: R/connectivity.R [hunk 1]
```diff
@@ -203,22 +203,16 @@ extract_connectivity_df <- function(rois, json){
...
-      n = as.numeric(gsub("[^0-9.]", "", c))
-      n = n[!is.na(n)]
-    }else{
-      n = c(0,0)
-    }
-    d = data.frame(n)
-    rownames(d) = paste0(roi,c("_pre","_post"))
-    values = rbind(values,d)
+    d <-  data.frame(0,0)
+    colnames(d) <- paste0(roi,c(".pre",".post"))
```
Comment (jefferis) [inline R/connectivity.R]: @romainFr I would normally stick with the same column names although I don't think anyone is hung up on the previous ones at this point. Was there a reason for the change?

**Pull request merged**